src/systems/filecoin_mining/storage_mining/storage_miner_actor.id

```diff
@@ -148,8 +143,13 @@ type StorageMinerActorCode struct {
         faultType      sector.StorageFaultType
     )
 
-    _claimDealPayments(rt Runtime)
+    ClaimDealPaymentsForSector(
```

Comment:
I think this doesn't need to be a public method - it's subsumed by `Withdraw`?

Reply:
Potentially, we still need a way for StorageMinerActor to trigger payment updates in the BalanceTable. We should touch this up in a follow-up.